src/System.Globalization/tests/CultureInfo/CultureInfoDateTimeFormat.cs

```diff
@@ -37,6 +38,44 @@ public void PosTest2()
             Assert.Equal("HH|mm", myCultureInfo.DateTimeFormat.ShortTimePattern);
         }
 
+        [Theory]
+        [InlineData("en-US", "h:mm tt")]
+        public void TestShortTimePatternLocale(string locale, string expected)
+        {
+            CultureInfo myTestCulture = new CultureInfo(locale);
+            DateTimeFormatInfo dtf = myTestCulture.DateTimeFormat;
+            Assert.Equal(expected, dtf.ShortTimePattern);
+        }
+
+        [InlineData("fi-FI", "H:mm", "H.mm")]
```

Comment:
My point is that this test doesn't even run right now, right?  I thought if there is no [Fact] or [Theory] attribute, then xUnit doesn't even execute the method.